library/VTKExtensions/Readers/CMakeLists.txt

```diff
@@ -3,7 +3,16 @@ set(classes
   vtkF3DPostProcessFilter
   )
 
+if(F3D_MODULE_EXR)
+  find_package(OpenEXR REQUIRED)
```

Comment:
version check ?

Reply:
why?

Reply:
are we compatible with all existing version of OpenEXR ?

Reply:
It's hard to tell.  
Ubuntu 20.04 has version 2.3  
Should I try this one and set it as the minimum version if it works?

Reply:
Debian 10 has 2.2.1, lets try that ?